app/components/document_download_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="al-show-actions-box-downloads-container">
+  <% case files.size %>
+  <% when 2 %>
+    <div class="dropdown">
+      <button class="btn btn-outline-secondary btn-sm dropdown-toggle me-1" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+        <%= t("arclight.views.show.download.default") %>
+      </button>
+      <ul class="dropdown-menu">
+        <% files.each do |file| %>
+          <li><%= link_to(file.href, class: "dropdown-item", **@link_options) do %>
+            <%= icon_for(file.type) %> <%= dropdown_label(file) %>
+          <% end %>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+  <% when 1 %>
+    <% files.each do |file| %>
+      <%= link_to(file.href, class: "btn btn-outline-secondary btn-sm me-1", **@link_options) do %>
```

Comment:
It is a little unfortunate we need to override to whole component just to change a couple of classes.  
I might be tempted to add the `me-1` via a .css file, but then I see that you also change `btn-secondary` to `btn-outline-secondary` on line 5.
It could be helpful to note somewhere that this is the same as the AL Core file, with change to utility classnames.

Reply:
This is an ongoing challenge with components. I think a comment would help. It did get me thinking about whether it might be appropriate to break off a `DownloadButtonComponent` to make the override a bit more targeted.

Reply:
Targeted overrides make sense but from my reading of your changes, they occur inside and outside of the button, I think? I see `btn btn-secondary btn-sm dropdown-toggle` and `btn btn-secondary btn-sm` for the `link_to`

Up to you but I think a comment would be fine!

Reply:
Comment added.